int8 a  # number 1
int8 b  # number 2
---
int8 sum  # summation result